tests/conftest.py

```diff
@@ -546,3 +546,82 @@ def _dir_sample_patches(_sample_patch1, _sample_patch2, tmpdir_factory):
         shutil.copy(_sample_patch2, dir_path.joinpath(_sample_patch2.basename))
 
     return dir_path
+
+
+@pytest.fixture(scope="session")
+def _sample_crc_tile(tmpdir_factory):
+    """Docstring."""
```

Comment:
Need to add some description here

Reply:
@vqdang  this docstring needs some description.

Reply:
Addressed in https://github.com/TIA-Lab/tiatoolbox/pull/123/commits/16e246236228fb37ab4b215c31b3601817e46a08